submissions/m-ruslan/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,195 @@
+                <input type="checkbox" class="seats__input seats__input_hidden" id="seat-6-1">
+                <label class="seats__label seats__label_free" for="seat-6-1"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden" id="seat-6-2">
+                <label class="seats__label seats__label_free" for="seat-6-2"></label>
+                <input type="checkbox" class="seats__input seats__input_hidde seats__input_booked" id="seat-6-3">
+                <label class="seats__label seats__label_booked" for="seat-6-3"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden seats__input_booked" id="seat-6-4">
+                <label class="seats__label seats__label_booked" for="seat-6-4"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden seats__input_booked" id="seat-6-5">
+                <label class="seats__label seats__label_booked" for="seat-6-5"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden seats__input_booked" id="seat-6-6">
+                <label class="seats__label seats__label_booked" for="seat-6-6"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden" id="seat-6-7">
+                <label class="seats__label seats__label_free" for="seat-6-7"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden" id="seat-6-8">
+                <label class="seats__label seats__label_free" for="seat-6-8"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden" id="seat-6-9">
+                <label class="seats__label seats__label_free" for="seat-6-9"></label>
+                <input type="checkbox" class="seats__input seats__input_hidden" id="seat-6-10">
+                <label class="seats__label seats__label_free" for="seat-6-10"></label>
+                <div class="seats__row">6</div>
+
+                <!--empty row-->
+                <div class="seats__emptyRow"></div>
+
+                <!--legend-->
+                <div class="seats__label seats__label_legendFree"></div>
+                <div class="seats__legend">Free</div>
+
+                <div class="seats__label seats__label_legendBooked"></div>
+                <div class="seats__legend">Booked</div>
+
+                <div class="seats__label seats__label_legendSelected"></div>
+                <div class="seats__legend">Selected</div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <h2 class="footer__header">Best Cinema LLC</h2>
+        <ul class="footer__list">
+
+            <li>
+                <address>13 Center Street, Capital City</address>
+            </li>
+            <li>tel: <a href="tel:+980991516171">+980991516171</a></li>
+            <li>email: <a href="mailto:bestcinema@gmail.com">bestcinema@gmail.com</a></li>
+        </ul>
+    </footer>
+</body>
+
+</html>
```

Comment:
Add an empty line at the end of each file.

Reply:
Fixed)